lib/ain-evm/src/evm.rs

```diff
@@ -143,6 +154,22 @@ impl EVMServices {
 
         let mut executor = AinExecutor::new(&mut backend);
 
+        if current_block_number == U256::zero() {
+            let CounterContractInfo {
+                address,
+                storage,
+                bytecode,
+            } = EVMServices::counter_contract().unwrap();
+            executor
+                .deploy_contract(address, bytecode, storage)
+                .unwrap();
+        } else {
+            let CounterContractInfo {
+                address, storage, ..
+            } = EVMServices::counter_contract().unwrap();
```

Comment:
Likewise over here. If contract deployment was unsuccessful with deploy_contract, error should be propagated and node state (if shutting down) should transit more elegantly.